Deeploy/TilingExtension/CodeTransformationPasses/TilingVariableReplacement.py

```
@@ -24,192 +24,207 @@
     @property
-    def prefix(self):
-        return self._prefix + f"{self._name}_" + self.targetMemLevel + "_"
+    def arenaName(self):
+        return f"{Tiler.arenaName}_{self.targetMemLevel}"
 
-    def _dereferencePointer(self, nodes: List[Node], name: str) -> List[Node]:
-        instanceIdxs = [idx for idx, node in enumerate(nodes) if isinstance(node, Expression) and node.text == name]
+    def _arenaAllocate(self, ctxt: NetworkContext, buffer: VariableBuffer, offset: int) -> VariableBuffer:
+        arena = ctxt.lookup(self.arenaName)
+        buffer.allocTemplate = NodeTemplate(" \
+        ${type.typeName} ${name} = (${type.typeName}) " + f"((char*){str(arena._instance)} + {offset});")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)